rocketpy/plots/tank_plots.py

```diff
@@ -1,3 +1,9 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.animation import FuncAnimation
+from matplotlib.patches import Polygon, Rectangle
```

Comment:
Same comment about unused imports. Suggestion:

```suggestion
import matplotlib.pyplot as plt
import numpy as np
from matplotlib.patches import Polygon
```